sdk/storage/azure-storage-blob/azure/storage/blob/_models.py

```diff
@@ -1014,6 +1018,42 @@ def from_string(cls, permission):
         return parsed
 
 
+class ContainerCpkScopeInfo(ContainerCpkScopeInfoGen):
+    """This scope is then used implicitly for all future writes within the container,
+    but can be overridden per-request via explicit request headers.
+
+    :param default_encryption_scope: Optional.  Version 2019-02-02 and later.
```

Comment:
> 2019-02-02 [](start = 55, length = 11)

shouldn't the version be 2019-07-07 instead?

Reply:
Yeah, good catch. I forgot to check when copy paste from generated code